test/windows/UnitTests.cpp

```diff
@@ -4346,6 +4349,67 @@ Error code: Wsl/Service/RegisterDistro/WSL_E_DISTRIBUTION_NAME_NEEDED\r\n";
             VERIFY_IS_FALSE(std::filesystem::exists(basePath));
         }
 
+        // Distribution with overridden default location but without explicit name (should use GUID)
+        {
+            auto cleanup = wil::scope_exit_log(
+                WI_DIAGNOSTICS_INFO, []() {
+                    auto distros = wsl::windows::common::SvcComm().EnumerateDistributions();
+                    for (const auto& distro : distros)
+                    {
+                        auto name = distro.DistributionName.get();
+                        if (wcsstr(name, L"Ubuntu") != nullptr)
+                        {
+                            std::wstring cmd = std::wstring(L"--unregister ") + name;
+                            LxsstuLaunchWsl(cmd.c_str());
+                            break;

```

Comment:
Let's use hardcoded name for the test distributions here, otherwise this might conflicts with pre-existing distributions and I'd rather not have the tests remove my distributions when I run them :)

Reply:
`CreateTarFromManifest()` can be used to generate a .tar with a specific default name